lib/archethic/contracts/interpreter.ex

```diff
@@ -822,10 +822,22 @@ defmodule Archethic.Contracts.Interpreter do
         "interval" ->
           [{{:atom, "at"}, interval}] = opts
 
+          # Extended Mode Signifies whether use CRON Extended Mode or Not while running SC Intervals
+          extended_mode =
+            case length(String.split(interval, " ", trim: true)) do
+              5 -> false
+              _ -> true
+            end
+
           Map.update!(
             acc,
             :contract,
-            &Contract.add_trigger(&1, :interval, [at: interval], actions)
+            &Contract.add_trigger(
```

Comment:
We want to avoid SC to use extended as it's irrelevant in the context of blockchain networks. 
No tx would be able to be validated in 1s with a significant number of nodes + network latency.
So I think we should pass directly in the worker the flag as false

Reply:
Oh okay, I also thought of the same in start but thought it is a added feature so implemented that way. Will make the changes to do simple no additional opts. 

Also there are not many checks on SC cron phrase, is it fine or will we add more restrict checks later

Reply:
There is check
https://github.com/archethic-foundation/archethic-node/blob/58491b69f566350dfae7b49690816f063faf28ca/lib/archethic/contracts/interpreter.ex#L995-L1003